starter-code/src/functions-and-arrays.js

```diff
@@ -11,14 +23,42 @@ var words = [
   'crackpot'
 ];
 
+function longestWord(list) {
+  var currentLongest = "";  
+  for (var i = 0; i < words.length; i++) {
+    if(list[i].length > currentLongest.length) {
+      currentLongest = list[i];
+    }
+  }
+  return currentLongest;
+}
+longestWord(words);
+
 // Calculating a Sum
 
 var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
+
+function add(list) {
+  for (var sum=0, i=0; i < numbers.length; i++) {
+    sum += list[i];
+  }
+  return sum;
+}
+add(numbers);
+
 // Calculate the Average
 
 var numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
 
+function avg(list) {
+  for (var sum=0, i=0; i < numbersAvg.length; i++) {
+    sum += list[i];
+  }
```

Comment:
This is a nice scenario where you could use the above add function to avoid repeating code